src/routes/companies/index.ts

```diff
@@ -0,0 +1,28 @@
+import { Router } from 'express';
+import cheerio from 'cheerio';
+import STRINGS from '../../constants/strings';
+
+import Scraper from '../../services/scraper';
+
+import getCompaniesFromOsloBors from './getCompaniesFromOsloBors';
+
+const router: Router = Router();
+const scraper = Scraper({ webscraper: cheerio });
+
+router.get('/:year', async (req, res, next) => {
+  try {
+    const $ = await scraper.getHTML(`${STRINGS.OSBX_URL}${req.params.year}`);
+    const table = $('tbody');
```

Comment:
If possible, this should be moved into the `getCompaniesFromOsloBors´ function, as it is quite specific business logic and should be split from the Controller code. 

The `catch` clause could be used to differ between the types of error thrown and return the correct response based on the error.

Reply:
Solved by b7f4bb3